### PR TITLE
component bottom sheets

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

File: src/bottomSheets/bottomSheets.styles.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.content {
+  background-color: $default-bg-color;
+  justify-content: center;
+}
+
+.separator {
+  border: 0;
+  border-bottom: solid 1px $default-grey-color;
```
Comment (fabienjuif) [inline src/bottomSheets/bottomSheets.styles.scss]: em

File: src/bottomSheets/bottomSheets.styles.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  left: 0;
+  margin: 0 2em;
+  position: fixed;
+  right: 0;
+}
+
+.sheets {
+  align-items: center;
+  background-color: $default-bg-color;
+  box-shadow: 0 0 5px 0 $font-color-dark;
```
Comment (fabienjuif) [inline src/bottomSheets/bottomSheets.styles.scss]: em

File: src/bottomSheets/bottomSheets.styles.scss (hunk 1 above)
Comment (guillaumecrespel) [reply src/bottomSheets/bottomSheets.styles.scss]: it's not relative, jsut one px it's good

File: examples/sagas/index.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
-import { takeLatest, takeEvery, put } from 'redux-saga/effects'
+import { takeLatest } from 'redux-saga/effects'
 import * as router from './model/router'
 
+const action = to => `BTN_CLICKED_MENU-ITEM-${to}`
```
Comment (fabienjuif) [inline examples/sagas/index.js]: `btnClick `?

File: examples/redux/ui/ui.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from 'redux'
 import tabs from './tabs'
 import { reducer as popover } from '../../../src/popover'
+import { reducer as bottomSheets } from '../../../src/bottomSheets'
 
 export default combineReducers({
   tabs,
+  bottomSheets,
```
Comment (fabienjuif) [inline examples/redux/ui/ui.js]: respect alphabetic order ?

File: examples/components/bottomSheets/awesome/detail/detail.styles.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+@import '~styles/theme';
+
+
```
Comment (fabienjuif) [inline examples/components/bottomSheets/awesome/detail/detail.styles.scss]: only one empty line

File: examples/sagas/index.js (hunk 3 above)
Comment (guillaumecrespel) [reply examples/sagas/index.js]: 👍

File: examples/redux/ui/ui.js (hunk 4 above)
Comment (guillaumecrespel) [reply examples/redux/ui/ui.js]: oké

File: examples/components/bottomSheets/awesome/detail/detail.styles.scss (hunk 5 above)
Comment (guillaumecrespel) [reply examples/components/bottomSheets/awesome/detail/detail.styles.scss]: 👍

File: src/bottomSheets/bottomSheets.styles.scss (hunk 2 above)
Comment (guillaumecrespel) [reply src/bottomSheets/bottomSheets.styles.scss]: 👍

**Pull request merged**